### PR TITLE
Log collector changes

### PR DESCRIPTION
**Pull request by BerniVarga**
Log-collector: https://github.com/Financial-Times/log-collector

Log-filter: https://github.com/Financial-Times/coco-logfilter/pull/41

### CODE REVIEW

**Review by kovacshuni — APPROVED** [review]

File: ansible/templates/cluster-template.yaml.j2 [hunk 1]
```diff
@@ -700,43 +703,46 @@ worker:
...
+            export BUCKET_NAME=$(etcdctl_cmd get /ft/config/splunk-forwarder/bucket_name); \
+            export AWS_REGION=$(etcdctl_cmd get /ft/config/splunk-forwarder/aws_region); \
+            export AWS_ACCESS_KEY_ID=$(etcdctl_cmd get /ft/config/splunk-forwarder/aws_access_key_id); \
+            export AWS_SECRET_ACCESS_KEY=$(etcdctl_cmd get /ft/config/splunk-forwarder/aws_secret_access_key); \
             journalctl -a -f --since=now --output=json \
-            | docker run -i --log-driver=none -e=ENV=$ENV --rm --name %p-filter_$(uuidgen) --memory="256m" coco/coco-logfilter:$DOCKER_LOGFILTER_VERSION \
-            | docker run -i --log-driver=none --rm --name %p-http_$(uuidgen) --memory="256m" -e="FORWARD_URL=$FORWARD_URL" -e="ENV=$ENV" -e="HOSTNAME=$(hostname -s)" -e="WORKERS=8" -e="BUFFER=256" -e="TOKEN=$TOKEN" -e="BATCHSIZE=$BATCHSIZE" -e="BATCHTIMER=5" coco/coco-splunk-http-forwarder:$DOCKER_FORWARDER_VERSION'
+            | docker run -i --log-driver=none -e=ENV=$ENV -e="DNS_ADDRESS=$DNS_ADDRESS" --rm --name %p-filter_$(uuidgen) --memory="256m" coco/coco-logfilter:$DOCKER_LOGFILTER_VERSION \
```
Comment (tamas-molnar) [inline ansible/templates/cluster-template.yaml.j2:742]: What does `DNS_ADDRESS` refer to?

File: ansible/templates/cluster-template.yaml.j2 [hunk 2]
```diff
@@ -279,42 +279,45 @@ controller:
...
       [Unit]
-      Description=Splunk forwarder
+      Description=Log S3 collector
       Wants=kubelet.service docker.service cfn-etcd-environment.service
       After=kubelet.service
       [Service]
       EnvironmentFile=/etc/etcd-environment
-      Environment="DOCKER_FORWARDER_VERSION=1.0.4"
-      Environment="DOCKER_LOGFILTER_VERSION=1.2.2-k8s"
+      Environment="DOCKER_LOG_COLLECTOR_VERSION=1.0.0-fd-rc5"
```
Comment (sbuliarca) [inline ansible/templates/cluster-template.yaml.j2]: If this was tested, please release final versions before the actual merge.

File: ansible/templates/cluster-template.yaml.j2 (hunk 1 above)
Comment (BerniVarga) [reply ansible/templates/cluster-template.yaml.j2:742]: The public dns name of the cluster - without the region level. 

It is used to deduce whether the cluster is actively serving responses, by verifying if the environment name can be found among the cnames for the original dns name. 

For example, if the dns_address is `staging-delivery.ft.com`, this evaluates to a `staging-eu-delivery.ft.com` cname, which has the `staging-eu` environment, so it serves traffic.

File: README.md [hunk 3]
```diff
@@ -61,12 +61,17 @@ kubectl create clusterrolebinding add-on-cluster-admin --clusterrole=cluster-adm
 4. [Just for UPP Clusters] Create/ amend the app-configs for the [upp-global-configs](https://github.com/Financial-Times/upp-global-configs/tree/master/helm/upp-global-configs/app-configs) repository. Build and deploy the global config to the new environment using this [Jenkins Job](https://upp-k8s-jenkins.in.ft.com/job/k8s-deployment/job/apps-deployment/job/upp-global-configs-auto-deploy/)
 5. [Restore](#restore-k8s-config) the config from a S3 backup or synchronize the cluster with an already existing cluster to deploy all the applications using this [Jenkins Job](https://upp-k8s-jenkins.in.ft.com/job/k8s-deployment/job/utils/job/diff-between-envs/).
 6. Connect through SSH to one of the etcd servers and use the command “etcdctl mk <key> <value>” to introduce the following etcd keys needed for forwarding the logs to splunk
+
 ```
-/ft/config/environment_tag
+/ft/config/public_address
```
Comment (sbuliarca) [inline README.md]: It would be good to know what to set these keys to

Comment (BerniVarga) [reply README.md]: I put some examples here.

**Review by sbuliarca — APPROVED** [review]

**Review by bsocaciu — APPROVED** [review]

**Review by tosan88 — APPROVED** [review]

**Pull request merged**